bootstrap/src/main/java/com/facebook/airlift/bootstrap/Bootstrap.java

```diff
@@ -138,9 +139,9 @@ public Bootstrap quiet()
         return this;
     }
 
-    public Bootstrap strictConfig()
+    public Bootstrap noStrictConfig()
```

Comment:
maybe adding a javadoc comment explaining where this would be useful might be a good idea.